packages-content-model/roosterjs-content-model-api/lib/modelApi/selection/adjustTrailingSpaceSelection.ts

```diff
@@ -0,0 +1,63 @@
+import { createText } from 'roosterjs-content-model-dom';
+import { iterateSelections } from 'roosterjs-content-model-core';
+import type {
+    ContentModelDocument,
+    ContentModelSegment,
+    ContentModelText,
+} from 'roosterjs-content-model-types';
+
+/**
+ * @internal
+ */
+export function adjustTrailingSpaceSelection(model: ContentModelDocument) {
```

Comment:
Given that this function is just for adjusting space selection after insert link or toggle underline, it is not a very common function. We need to add comment to explain it. So that in case other people want to use this function in the future, they need to understand what this function is doing.